### PR TITLE
Prepare release 4.0.5

### PR DESCRIPTION
**Pull request by mkleen**
## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

File: blackbox/docs/appendices/release-notes/4.0.5.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+   :local:
+
+
+See the :ref:`version_4.0.0` release notes for a full list of changes in the
+4.0 series.
+
+Changes
+=======
+
+- Improved the help section of the admin-ui and added Spanish translations.
```
Comment (kovrus) [inline blackbox/docs/appendices/release-notes/4.0.5.rst]: The hotfix should probably contain only fixes? Also, can we maybe rephrase a bit the part about the Spanish translation, I had to check the Admin-UI release notes to understand what was actually translated/fixed. https://github.com/crate/crate-admin/commit/7c7763ba9c4bf83d2bf1720a1579e6ea4cb1692a#diff-bf59ca62b9676f8b8e05f808bdd46628L132

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 2]
```diff
@@ -41,26 +41,7 @@ Unreleased Changes
 Changes
 =======
 
-- Improved the help section of the admin-ui and added Spanish translations.
```
Comment (kovrus) [inline blackbox/docs/appendices/release-notes/unreleased.rst]: If it was a change, why then we removed it from here?

File: blackbox/docs/appendices/release-notes/4.0.5.rst (hunk 1 above)
Comment (seut) [reply blackbox/docs/appendices/release-notes/4.0.5.rst]: yes this is a bit of a grey area as the related commit contains fixes and improvements. I'd suggest to move this entry also to the `Fixes` and rephrase it to `improved Spanish translations`.

Comment (mkleen) [reply blackbox/docs/appendices/release-notes/4.0.5.rst]: > yes this is a bit of a grey area as the related commit contains fixes and improvements. I'd suggest to move this entry also to the `Fixes` and rephrase it to `improved Spanish translations`.

Ok, do we just do this in this release pr or also in the admin-ui commit before ?

File: blackbox/docs/appendices/release-notes/4.0.5.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+Changes
+=======
+
+None
+
+Fixes
+=====
+
+- Improved the help section including the spanish translations.
```
Comment (seut) [inline blackbox/docs/appendices/release-notes/4.0.5.rst]:
```suggestion
- Improved the help section of the admin-ui including the spanish translations.
```

File: blackbox/docs/appendices/release-notes/4.0.5.rst [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+
+See the :ref:`version_4.0.0` release notes for a full list of changes in the
+4.0 series.
+
+Changes
```
Comment (seut) [inline blackbox/docs/appendices/release-notes/4.0.5.rst]: pls remove, we have no changes and thus this empty section doesn't make sense
